### ISSUE TITLE
QT Client: don't show some information when not available

### ISSUE THREAD

**Issue by amirmahdifard** [BearWare/TeamTalk5#1567]
### Description

let me example 2 things to show this: so y it should say something if it is not available for a user, 1: in online users dialogue, it is like this.
ID: 158, Nickname: Audio Player, Status message: , Username: player, Channel: /Audio Box/, IP address: 103.152.67.161, Version: 5.9.1  collapsed  row 1  ID  column 1  level 1
this audio player doesn't have any statius message, so y it is saying status message if it doesn't have? it should only say status message if the user have status message and same with other information, as Classic client: in user accounts list dialogue, it is like this.
Username: amirmahdifard, Password: blaablaablaa, Type: Administrator, Note: , Initial channel: , Modified: 2022/07/02 11:36  row 1  Username  column 1  level 1
in this case, my user doesn't have any note, and doesn't have any Initial channel too: so it shouldn't say the title of them if the user doesn't have, and by the way, don't forget to also remove those useless information that i've reported in my previous ishou: QT Client vertion: 5.11

### Application

- [X] qtTeamTalk
- [ ] TeamTalkAndroid
- [ ] iTeamTalk
- [ ] TeamTalkServer

**Comment by CoBC** [comment]
@bear101 The only way I see to fix it is to check one by one available information, make one string for each information and than build the final QString put in accessible text. This, however, requires a lot of changes especially for translations, so I don't know if benefit is greater than require changes. What do you think?

**Comment by amirmahdifard** [comment]
@CoBC can you please answer me? i feel when i'm commenting on something in github, people are not getting my comment.

**Comment by bear101** [comment]
@CoBC it's up to you. If Qt later changes the behaviour then it's a bit wasted.

**Comment by amirmahdifard** [comment]
it doesn't that matter to be fixed, it doesn't stop us from working with the app, you can fix the other problem that i've reported: leave this one behind and add triviel  lable.

**Comment by amirmahdifard** [comment]
@CoBC @bear101 this problem is doesn't matter, the other problems are more matters to be fixed. please add trival lable or i can close it.

**Comment by amirmahdifard** [comment]
closing this for now because there is other problems that needs fixing.
